src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -789,7 +791,7 @@
 
   "block.minecolonies.blockpaperwall": "Paper wall",
 
-  "com.minecolonies.coremod.gui.workerhuts.buildprio": "Delivery Priority:",
+  "com.minecolonies.coremod.gui.workerhuts.buildprio": "Gathering Priority:",
```

Comment:
i suggest prefixing this with "Item", confusing otherwise

Reply:
too big for the GUI